README.rst

```diff
@@ -227,6 +227,31 @@ Installing the latest master branch of Salt:
   curl -L https://bootstrap.saltstack.com | sudo sh -s -- git master
 
 
+Install on Windows
+~~~~~~~~~~~~~~~~~~
+
+Using ``PowerShell`` to install latest development version from GitHub:
+
+.. code:: console
+
+  Invoke-WebRequest `
+    -Uri https://winbootstrap.saltstack.com
+    -OutFile C:\Temp\bootstrap-salt.ps1
+  Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser
+  C:\Temp\bootstrap-salt.ps1
+  Set-ExecutionPolicy -ExecutionPolicy Undefined -Scope CurrentUser
+
+
+Using ``cygwin`` to install latest development version from GitHub:
```

Comment:
to install the development version, it's https://winbootstrap.saltstack.com/develop  without the /develop you get the stable version.

also, it's not "from Github" if it's not a github url is it?

Reply:
I would recommend using the stable version in most circumstances.

Reply:
Thanks, I have updated commit  for these comments.